node/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "azure-pipelines-task-lib",
-  "version": "3.0.0-preview",
+  "version": "3.0.1-preview",
   "lockfileVersion": 1,
```

Comment:
FYI, I ran `npm install npm && npm install` and there were a lot more changes in this file. All the sliding dependencies like `^2.2.2` were locked onto a particular version.

Is this due to using the latest npm? Or is there some npm configuration we should be using?